.github/SECURITY.md

```diff
@@ -0,0 +1,26 @@
+# Security Policy
```

Comment:
Let's link to that file from the README.
Like: https://github.com/cakephp/cakephp#security